src/tinypeel/tinypeel.py

```diff
@@ -416,28 +562,46 @@ def getArgs():
 
 def main():
     args = getArgs()
```

Comment:
Please also add

```
    if args.start_snp:
            args.start_snp -= 1
    if args.stop_snp:
            args.stop_snp -= 1
```

This is modified from `tinyhouse.InputOutput.parseArgs`, but now we also need it here.